src/app/api/v1/slack/rotation/on-holiday-post/route.ts

```diff
@@ -0,0 +1,57 @@
+import { PostgresClient } from 'utils/database';
+import { NextRequest, NextResponse } from 'next/server';
+import {
+    getSlackMessage,
+    SlackResponseType,
+    sanitizeSlackText,
+    parsePayloadFromRequest,
+} from 'utils/slack';
+
+export async function POST(req: NextRequest) {
+    try {
+        const parsedPayload = await parsePayloadFromRequest(req);
+        const { text, team_domain } = parsedPayload;
+
+        // Extract and sanitize Slack ID, convert to uppercase, else it breaks
+        const slackIdMatch = text.match(/<@([A-Z0-9]+)>/);
+        const slackId = slackIdMatch ? sanitizeSlackText(slackIdMatch[1]).toUpperCase() : null;
```

Comment:
how about multiple users? something like

`/rr-skip some rotation @pimienta @someoneelse` ?
@nenedac

Reply:
Sure thing, I'll work on that on the afternoon